tests/components/accuweather/test_sensor.py

```diff
@@ -42,12 +49,12 @@
 )
 
 
-async def test_sensor_without_forecast(
+async def test_sensor(
```

Comment:
Snapshots would be a nice improvment for this test

Reply:
As you wish 😄